resources/validations/validate_with_schematron.sh

```diff
@@ -0,0 +1,38 @@
+if [ ! -e  "$1" ]; then
```

Comment:
Shellcheck rightfully recommends we explicitly set a shebang to know what shell you are using.

I know you use `zsh`, and it is backwards compatible with `bash` and the latter is more common, but up to you.

```suggestion
#!/usr/bin/env bash

if [ ! -e  "$1" ]; then
```

Reply:
updated